Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,53 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<Entities.ContactPoliciesSettings> GetBasicContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select cp.IdUser, cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
+                from [User] u
+                left join [UserShippingLimit] cp on u.IdUser = cp.IdUser and cp.Enabled = 1
+                where u.Email = @Email;";
+
+            var queryParams = new { Email = accountName };
+            return await connection.QueryFirstOrDefaultAsync<Entities.ContactPoliciesSettings>(query, queryParams);
+        }
+
+        public async Task<bool> UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesToInsert)
+        {
+
+            using var connection = await _databaseConnectionFactory.GetConnection();
+
+            using (var transaction = connection.BeginTransaction())
```

Comment:
En algún momento tuve problemas con `Dapper` + `BeginTransaction`, terminé usando `TransactionScope`. Te aviso por si, al momento de probar, la transaction (rollback) no funciona como esperas. Avisame cualquier cosa.

Reply:
At the moment is doing the rollback very well, i test that at beginning of implementation

Reply:
Great!, I am marking it as resolved...